onsp_co2/views/carbon_factor.xml

```diff
@@ -35,7 +35,55 @@
                     <sheet>
                         <div class="oe_button_box" name="button_box">
                             <button type="object" name="action_see_child_ids" class="oe_stat_button" icon="fa-sitemap">
-                                <field name="child_qty" widget="statinfo" string="Child Factors" />
+                                <div class="o_field_widget o_stat_info">
```

Comment:
Is this worth for some plurals? :joy: 


Anyway, maybe change "Child Factors" and "Categorys"

Reply:
It was asked by BCH so i only execute the order xDDDDD